src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -364,10 +366,22 @@ private com.exactpro.th2.common.grpc.Event toProto(
                 .setStatus(getAggregatedStatus().eventStatus)
                 .setBody(ByteString.copyFrom(buildBody()));
         if (parentId != null) {
+            if (!Objects.equals(parentId.getBookName(), eventId.getBookName())) {
```

Comment:
I think we should check the scope as well. Otherwise, it won't be possible to display parent and child events at the same time